draft-js-mention-plugin/src/MentionSuggestions/index.js

```diff
@@ -206,6 +208,9 @@ export default class MentionSuggestions extends Component {
   };
 
   onMentionSelect = (mention) => {
+    if (!mention) {
```

Comment:
user didn't select mention from mentionsuggestions, instead they can just by typing @xxx (invalidate mentions) then hit enter .. in that case mention will be undefined..